atat_provisioning_api.yaml

```
@@ -0,0 +1,269 @@
+              $ref: '#/components/schemas/ProvisioningRequest'
+    options:
+      tags:
+        - cors
+      description: CORS headers
+      responses:
+        '200':
+          $ref: '#/components/responses/CorsHeaders'
+components:
+  responses:
+    CorsHeaders:
+      description: Default response for CORS method
+      headers:
+        Access-Control-Allow-Origin:
+          schema:
+            type: string
+        Access-Control-Allow-Methods:
+          schema:
+            type: string
+        Access-Control-Allow-Headers:
+          schema:
+            type: string
+      content: {}
+  schemas:
+    Error:
+      type: object
+      additionalProperties: false
+      description: Generic error model
+      properties:
+        code:
+          type: string
+          enum: [INVALID_INPUT, OTHER]
+        message:
+          type: string
+      required:
+        - code
+        - message
+    ValidationError:
+      allOf:
+        - $ref: '#/components/schemas/Error'
+        - type: object
+          additionalProperties: false
+          required:
+            - error_map
+          properties:
+            error_map:
+              type: object
+              description: 'Maps form input IDs to validation error messages so that clients can display in-line errors'
+    ProvisioningRequest:
+      description: Represents a generic request for provisioning cloud resources via ATAT
+      type: object
+      additionalProperties: false
+      required:
+        - job_id
+        - user_id
+        - portfolio_id
+        - operation_type
+        - target_cloud
+        - target_network
+        - payload
+      properties:
+        job_id:
+          description: ID of the Provisioning Job assigned by DISA StoreFront Service Now
+          type: string
+        user_id:
+          description: ID of the Mission Owner in DISA StoreFront Service Now
+          type: string
+        portfolio_id:
+          description: ID of the CSP Portfolio assigned by DISA StoreFront Service Now (not to be confused with the ID generated by the CSP)
+          type: string
+        operation_type:
+          $ref: '#/components/schemas/OperationType'
+        target_cloud:
+          $ref: '#/components/schemas/TargetCloud'
+        target_network:
+          $ref: '#/components/schemas/TargetNetwork'
+        payload:
+          oneOf:
+            - $ref: '#/components/schemas/NewPortfolioPayload'
+            - $ref: '#/components/schemas/FundingSourcePayload'
+            - $ref: '#/components/schemas/OperatorPayload'
+        response:
+          description: Response from the CSP which will be set after a request has been processed
+          $ref: '#/components/schemas/CspResponse'
+          readOnly: true
+    OperationType:
+      description: Distinguishes type of operation that a user is performing
+      type: string
+      enum:
+        - ADD_PORTFOLIO
+        - ADD_OPERATORS
+        - ADD_FUNDING_SOURCE
+    TargetCloud:
```

As we discussed via call.  Unsure about meaning of _cloud_ vs. _csp_ terminology.

```suggestion
    TargetCloudServiceProvider:
```